backend/src/data/migrations/20220810080145_add_users_associations.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex';
+
+const TableName = {
+  USERS_TO_GROUPS: 'users_to_groups',
+  USERS: 'users',
+} as const;
+
+const ColumnName = {
+  ID: 'id',
+  GROUP_ID: 'group_id',
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable(TableName.USERS, (table) => {
+    table
+      .integer(ColumnName.GROUP_ID)
+      .references(ColumnName.ID)
+      .inTable(TableName.USERS_TO_GROUPS);
+  });
+}
+
+async function down(knex: Knex): Promise<void> {
+  await knex.schema.alterTable(TableName.USERS, (table) => {
+    table.dropColumn(ColumnName.GROUP_ID);
+  });
```

Comment:
As for now we have users_to_groups table to make connection on users to their groups,, and that associations are already declared (look for `20220809045744_add_users_to_groups_associations.ts` made by Alina), so we don't need `group_id` column in `users` table at all - that means you don't need to add this migration
```suggestion
  
```